app/server/src/features/events/questions/resolvers.ts

```diff
@@ -27,21 +27,35 @@ export const resolvers: Resolvers = {
                     node: formattedQuestion,
                     cursor: formattedQuestion.createdAt.getTime().toString(),
                 };
-                // TODO: #QQRedesign delete once code complete
                 ctx.pubsub.publish({
-                    topic: 'questionCRUD',
+                    topic: 'questionCreated',
                     payload: {
-                        questionCRUD: { operationType: 'CREATE', edge } as QuestionOperation,
+                        questionCreated: { edge },
                     },
                 });
+
+                return edge;
+            });
+        },
+        async deleteQuestion(parent, args, ctx, info) {
+            return runMutation(async () => {
+                if (!ctx.viewer.id) throw new Error(errors.noLogin);
+                // TODO: Add check to ensure question is not currently enqueued (or also remove from queue/record)
```

Comment:
I would maybe add this check now since it's simple.  I don't think moderators would do this purposefully, but  accidentally. I think you can just tell if it's enqueued if it's `position !== - 1`?  So you can make a method like `isInQueue`.

Reply:
Also, as part of the selection for delete you might be able to say `position: -1` so it won't match a question whose position is not -1?  Just a thought, not sure if that's the best way to go about it.

Reply:
Resolved in other convos.